### ISSUE TITLE
"Include Timestamp" text in share dialogue hardly readable in dark mode

### ISSUE THREAD

**Issue by MariusAlbrecht** [joschahenningsen/TUM-Live#1167]
**Describe the bug**
"Include Timestamp" text in share dialogue is dark blue and gets even darker on hover. This makes the text practically unreadable (to me)
contrast ratio is 1.75 which is bad (violates [WCAG 2.1 AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) e.g.)

**To Reproduce**
1. Open any stream
2. open the share dialogue
3. check the box so that the text turns blue
4. optionally hover on the text

**Expected behavior**
High enough contrast to make text readable and accessible

**Screenshots**
![image](https://github.com/TUM-Dev/gocast/assets/64852250/9ad1eeb2-b180-4110-b2d9-9f5dc12fc5cd)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox 117.0.1
